src/diffusers/pipelines/audioldm2/modeling_audioldm2.py

```diff
@@ -108,6 +108,12 @@ def __init__(self, text_encoder_dim, text_encoder_1_dim, langauge_model_dim):
         self.sos_embed_1 = nn.Parameter(torch.ones(langauge_model_dim))
         self.eos_embed_1 = nn.Parameter(torch.ones(langauge_model_dim))
 
+        # learable positional embedding for vits encoder
+        if text_encoder_1_dim == 192:
```

Comment:
Can we add an additional arg to the init `use_learned_position_embedding` and perform the check based on whether it is `None` or not?